exes/mqtt-broadcaster/inc/mqtt_client.hpp

```diff
@@ -0,0 +1,71 @@
+#pragma once
+
+#include <openssl/ssl.h>
+#include <async_mqtt/buffer.hpp>
+#include <async_mqtt/endpoint.hpp>
+#include <async_mqtt/packet/qos.hpp>
+#include <async_mqtt/packet/v5_publish.hpp>
+#include <async_mqtt/predefined_underlying_layer.hpp>
+#include <async_mqtt/tls.hpp>
+#include <boost/asio/any_io_executor.hpp>
+#include <boost/asio/awaitable.hpp>
+#include <boost/asio/co_spawn.hpp>
+#include <boost/asio/connect.hpp>
+#include <boost/asio/detached.hpp>
+#include <boost/asio/io_context.hpp>
+#include <boost/asio/ip/tcp.hpp>
+#include <boost/asio/strand.hpp>
```

Comment:
This can be forward declared

Reply:
Newest version attempts to forward declare as much as possible.